disnake/ext/commands/interaction_bot_base.py

```diff
@@ -356,9 +356,11 @@ def remove_message_command(self, name: str) -> Optional[InvokableMessageCommand]
             return None
         return command
 
-    def get_slash_command(self, name: str) -> Optional[InvokableSlashCommand]:
-        """Get a :class:`.InvokableSlashCommand` from the internal list
-        of commands.
+    def get_slash_command(
+        self,
+        name: str
+    ) -> Optional[Union[InvokableSlashCommand, SubCommandGroup, SubCommand]]:
+        """Works like ``Bot.get_command``, but for slashes.
```

Comment:
```suggestion
        """Works like ``Bot.get_command``, but for slash commands.
```

I think "slash commands" is an appropriate way to refer to them, rather than using "slashes".